static/css/header.css

```
@@ -12,11 +12,23 @@ header {
 
 header .navbar .navbar-nav a {
     color: var(--text-color) !important;
+    display: flex;
+    align-items: center; /* Vertically align all items */
 }

```

I like this fix, definitely bothered me as well. However, with this implementation on mobile, the hamburger menu items are all aligned to the left (they were aligned to the center before). Personally, I like that aligned more, which thus would require this change 
```suggestion
header .navbar .navbar-nav a,
header .navbar .navbar-nav div {
  color: var(--text-color) !important;
  display: flex;
  align-items: center; /* Vertically align all items */
  justify-content: center /* Horizontally align all items */;
}
```

This ofc is up to @gurusabarish, however they see fit

---

Hey, not opposed to implementing this in the PR. Originally I only checked how this looked on Desktop, not mobile, but noticed it several weeks after the PR. I just was okay to live with it. When I'm back at a computer I'll see about trying to incorporate that change in to the PR.